src/main/java/core/basesyntax/MainApp.java

```diff
@@ -2,4 +2,25 @@
 
 public class MainApp {
 
+    public static void main(String[] args) {
+
+        Machine[] machines = new Machine[]{new Truck(), new Bulldozer(), new Excavator()};
+
+        for (Machine machine : machines) {
+            machine.doWork();
+        }
+
+        System.out.println();
+
+        for (Machine machine : machines) {
+            machine.isActive();
```

Comment:
better make this checking before stop working. Like this:
```
for (Machine machine : machines) {
    if(machine.isActive()) {
        machine.stopWork();
    }
}
```

Reply:
Have a sense. Thanks!